trulens_eval/examples/expositional/use_cases/vectra_hallucination_evaluation_model.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>This notebook seems similar to the one under <code>/models</code> - what is the distinction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/950/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>